src/pyhf/tensor/pytorch_backend.py

```diff
@@ -219,6 +224,9 @@ def simple_broadcast(self, *args):
             list of Tensors: The sequence broadcast together.
         """
 
+        args = [
+            arg if arg.size() != torch.Size([]) else arg.unsqueeze(0) for arg in args
+        ]
```

Comment:
There might be a more elegant way to check if unsqueezing to a 1-d tensor is needed, but PyTorch is a bit weird about callables for size and shape, given that [`.shape` is an alias for `.size()`](https://github.com/pytorch/pytorch/issues/5544#issuecomment-370166015).